src/main/java/mate/academy/spring/service/dto/mapping/impl/response/MovieSessionResponseMapper.java

```diff
@@ -14,9 +14,8 @@ public MovieSessionResponseDto toDto(MovieSession movieSession) {
         MovieSessionResponseDto movieSessionResponseDto = new MovieSessionResponseDto();
         movieSessionResponseDto.setId(movieSession.getId());
         movieSessionResponseDto.setMovieTitle(movieSession.getMovie().getTitle());
-        movieSessionResponseDto.setShowTime(movieSession.getShowTime()
-                                            .format(DateTimeFormatter
-                                                    .ofPattern("dd.MM.yyyy HH:mm")));
+        movieSessionResponseDto.setShowTime(movieSession
+                .getShowTime().format(DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm")));
```

Comment:
I think, it's better use PATTERN here instead of "dd.MM.yyyy HH:mm"

public static final String PATTERN = "dd.MM.yyyy HH:mm";